### PR TITLE
PIM parcel

### PR DESCRIPTION
**Pull request by hshaosf**
PIM parcel (block/lot) field

### CODE REVIEW

File: service/resources/arcgis.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            resp.status = falcon.HTTP_404
+            msg_error = jsend.error(
+                'Missing or invalid environment variable(s): '+", ".join(missing_env))
+            resp.body = json.dumps(msg_error)
+            return
+
+        response = {'message': 'parcels'}
+        if 'address' in req.params:
+            addr = usaddress.tag(req.params['address'])
+            options = {'returnGeometry':'false', 'returnSuggestions':False}
```
Comment (josh-chou) [inline service/resources/arcgis.py]: Kind of weird that one is string and the other is boolean

File: service/resources/arcgis.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                options['returnGeometry'] = req.params['returnGeometry']
+            parcels = self.pln_parcel(addr, options)
+            response = {'parcels': parcels}
+        resp.body = json.dumps(jsend.success(response))
+        resp.status = falcon.HTTP_200
+
+    def pln_parcel(self, addr, options=None):
+        """ get parcels from Planning ArcGIS """
+        parcels = {}
+        url = os.environ.get('PLN_ARCGIS_PARCEL')+'query'
```
Comment (josh-chou) [inline service/resources/arcgis.py]: This assumes that the env var ends with "/".  What happens when it doesn't?

Comment (hshaosf) [reply service/resources/arcgis.py]: commit bf10e31362024045178aaef0ac51197f7a4d3fa1

File: service/resources/arcgis.py (hunk 1 above)
Comment (hshaosf) [reply service/resources/arcgis.py]: commit 961212fb64fb4f346b8cfc94e2fab0cf577fa1a0

**Pull request merged**